libs/ledger-live-common/src/families/aptos/getTransactionStatus.test.ts

```diff
@@ -0,0 +1,283 @@
+  });
+
+  it("should return errors for InvalidAddress", async () => {
+    const account = createFixtureAccount();
+    const transaction = createTransaction();
+
+    transaction.amount = new BigNumber(2);
+    transaction.fees = new BigNumber(2);
+    transaction.recipient = "0x";
+
+    const result = await getTransactionStatus(account, transaction);
+
+    const expected = {
+      errors: {
+        recipient: new InvalidAddress(),
+      },
+      warnings: {},
+      estimatedFees: new BigNumber(2),
+      amount: new BigNumber(2),
+      totalSpent: new BigNumber(4),
+    };
+
+    expect(result).toEqual(expected);
+  });
+
+  it("should return errors for InvalidAddressBecauseDestinationIsAlsoSource", async () => {
+    const account = createFixtureAccount();
+    const transaction = createTransaction();
+
+    transaction.amount = new BigNumber(2);
+    transaction.fees = new BigNumber(2);
+    transaction.recipient = "0x" + "0".repeat(64);
+    account.freshAddress = "0x" + "0".repeat(64);
+
+    const result = await getTransactionStatus(account, transaction);
+
+    const expected = {
+      errors: {
+        recipient: new InvalidAddressBecauseDestinationIsAlsoSource(),
+      },
+      warnings: {},
+      estimatedFees: new BigNumber(2),
+      amount: new BigNumber(2),
+      totalSpent: new BigNumber(4),
+    };
+
+    expect(result).toEqual(expected);
+  });
+
+  it("should return errors for GasLessThanEstimate", async () => {
+    const account = createFixtureAccount();
+    const transaction = createTransaction();
+
+    transaction.amount = new BigNumber(2);
+    transaction.fees = new BigNumber(2);
+    transaction.recipient = "0x" + "0".repeat(64);
+
+    transaction.options.maxGasAmount = "50";
+    transaction.estimate.maxGasAmount = "100";
+
+    const result = await getTransactionStatus(account, transaction);
+
+    const expected = {
+      errors: {
+        maxGasAmount: new GasLessThanEstimate(),
+      },
+      warnings: {},
+      estimatedFees: new BigNumber(2),
+      amount: new BigNumber(2),
+      totalSpent: new BigNumber(4),
+    };
+
+    expect(result).toEqual(expected);
+  });
+
+  it("should return errors for GasLessThanEstimate", async () => {
+    const account = createFixtureAccount();
+    const transaction = createTransaction();
+
+    transaction.amount = new BigNumber(2);
+    transaction.fees = new BigNumber(2);
+    transaction.recipient = "0x" + "0".repeat(64);
+
+    transaction.options.gasUnitPrice = "50";
+    transaction.estimate.gasUnitPrice = "100";
+
+    const result = await getTransactionStatus(account, transaction);
+
+    const expected = {
+      errors: {
+        gasUnitPrice: new GasLessThanEstimate(),
+      },
+      warnings: {},
+      estimatedFees: new BigNumber(2),
+      amount: new BigNumber(2),
+      totalSpent: new BigNumber(4),
+    };
+
+    expect(result).toEqual(expected);
+  });
+
+  it("should return errors for SequenseNumberTooOldError", async () => {
```

Comment:
Could you edit the typo (sequense => sequence) in this PR ? 🙏

Reply:
ofc 😉